importer.js

```diff
@@ -0,0 +1,53 @@
+import fs from 'fs';
+import nodePath from 'path';
+import csvjson from 'csvjson';
+import { promisify } from 'util';
+const readFileAsync = promisify(fs.readFile);
+const readDirAsync = promisify(fs.readdir);
+
+class Importer {
+    constructor(dirWatcher) {
+        dirWatcher.on('changed', async (path) => {
+            /**
+             * Watching for file changes Syncrhonously
+             */
+            const syncData = this.importSync(path);
+            console.log(result)
```

Comment:
Please rename the variable you log to the console.

Reply:
Apologies, Updated with correct variable.